compiler/vm/vmprofiler.nim

```diff
@@ -57,5 +57,5 @@ proc dump*(conf: ConfigRef, pd: ProfileData): string =
       break
     result.add  "  " & align($int(infoMax.time * 1e6), 10) &
                        align($int(infoMax.count), 10) & "  " &
-                       conf.toFileLineCol(flMax) & "\n"
+                       toMsgFilename(conf, flMax.fileIndex) & "(" & $flMax.line & ")" & "\n"
```

Comment:
```suggestion
    result.toLocation(toMsgFilename(conf, flMax.fileIndex), flMax.line, 0)
    result.add "\n"
```
This ensures rendering of line information is implemented uniformly (through `toLocation`). For the procedure to be available, you need to import the `std/private/miscdollars` module.